aws/src/database/domain/dynamo_domain_objects.py

```diff
@@ -209,28 +209,44 @@ def from_data(cls, data: dict):
         return _dataclass_from_data(cls, data)
 
 
+@dataclass
+class AssetAddress:
+    addressLine1: str | None
+    addressLine2: str | None
+    addressLine3: str | None
+    addressLine4: str | None
+    postCode: str | None
+    postPreamble: str | None
+    uprn: str | None
+
+    @classmethod
+    def from_data(cls, data: Any):
+        return _dataclass_from_data(cls, data)
+
+
 @dataclass
 class Asset:
     id: str
     areaId: str | None
+    assetId: str | None
     patchId: str | None
     assetAddress: AssetAddress | None
     assetCharacteristics: dict | None
-    assetId: str | None
     assetLocation: dict | None
     assetManagement: dict | None
     assetType: str | None
     isActive: int | None
     rootAsset: str | None
     tenure: AssetTenure | None
+    rentGroup: str | None
```

Comment:
Why is rent group twice?

Reply:
Fixed in f7a4811aa95b3836a923dfa17e208f952a6d51b2